_search-plugins/hybrid-search.md

```diff
@@ -12,7 +12,7 @@ Introduced 2.11
 Hybrid search combines keyword and neural search to improve search relevance. To implement hybrid search, you need to set up a [search pipeline]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/index/) that runs at search time. The search pipeline you'll configure intercepts search results at an intermediate stage and applies the [`normalization_processor`]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/normalization-processor/) to them. The `normalization_processor` normalizes and combines the document scores from multiple query clauses, rescoring the documents according to the chosen normalization and combination techniques. 
 
 **PREREQUISITE**<br>
-Before using hybrid search, you must set up a text embedding model. For more information, see [Choosing a model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/integrating-ml-models/#choosing-a-model).
+To follow this example, you must set up a text embedding model. For more information, see [Choosing a model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/integrating-ml-models/#choosing-a-model).
```

Comment:
can we mention that having a embedding model is not required if you already have the embeddings for the text

Reply:
In this case, they can just directly set up hybrid search, correct? I added that sentence.

Reply:
no actually thats not the case. What we need to tell them is if you already have the embeddings directly ingest the embeddings and skip the ingest pipeline creation step.

Reply:
Sure. I revised the sentence.